api/src/pcapi/routes/native/v1/serialization/account.py

```diff
@@ -28,6 +28,15 @@
 from pcapi.utils.email import sanitize_email
 
 
+class TrustedDevice(BaseModel):
+    device_id: str
+    os: str | None
+    source: str | None
+
+    class Config:
+        alias_generator = to_camel
```

Comment:
En ajoutant un peu de configuration, on supprimera automatiquement les espaces dans le "validateur" d'entrée de la route. Ainsi, on n'aura plus besoin de le faire dans `core.users.api.save_trusted_device()`.

```suggestion
        alias_generator = to_camel
        anystr_strip_whitespace = True
```